docs/data/pages.ts

```diff
@@ -1,8 +1,15 @@
 const pages = [
   {
-    pathname: '/x/advanced-components',
-    title: 'Overview',
+    pathname: '/x/getting-started/overview',
+    scopePathnames: ['/x/getting-started/overview'],
```

Comment:
```suggestion
    scopePathnames: ['/x/getting-started/'],
```

to avoid menu collapse after changing the page:

https://user-images.githubusercontent.com/13808724/166507986-6fea8a1a-8ce7-4425-9c66-bd00023206d0.mov

Reply:
Excellent! I couldn't figure out why that was happening. 😅